lib/OpenQA/WebAPI/Controller/API/V1/JobTemplate.pm

```diff
@@ -155,7 +155,7 @@ sub create {
     }
     else {
         $json->{id} = $id;
-        $self->emit_event('openqa_jobtemplate_create', {id => $id});
+        $self->emit_event('openqa_jobtemplate_create', {id => $id, %{$self->req->body_params->to_hash}})
```

Comment:
Not knowing anything about how the `openqa_jobtemplate_create` event might get used, is this a possible attack vector? If `$self->req->body_params->to_hash` contained `id=123` it would redefine the `id => $id` value. Perhaps `{%{$self->req->body_params->to_hash}, id => $id}` would be appropriate to guarantee that the value can't be changed with POST parameters?